### PR TITLE
MAKE-792 filter perf results by variant

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: rest/data/interface.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ type Connector interface {
 	FindPerformanceResultById(string) (*model.APIPerformanceResult, error)
 	RemovePerformanceResultById(string) (int, error)
 	FindPerformanceResultsByTaskId(string, util.TimeRange, ...string) ([]model.APIPerformanceResult, error)
-	FindPerformanceResultsByTaskName(string, util.TimeRange, int, ...string) ([]model.APIPerformanceResult, error)
+	FindPerformanceResultsByTaskName(string, string, util.TimeRange, int, ...string) ([]model.APIPerformanceResult, error)
```
Comment (tychoish) [inline rest/data/interface.go]: Should we change the method name to match?

Comment (john-m-liu) [reply rest/data/interface.go]: I think the method name is still correct since the only required info is the task name

**Comment by tychoish** [comment]
I think the tests should pass?

**Comment by john-m-liu** [comment]
It looks flaky to me. I didn't change anything related to that though it passes on the base commit. I restarted it

**Pull request merged**